lib/little-loader.js

```diff
@@ -51,27 +55,30 @@
    * @param {Object}    context   (Optional) Callback context
    * @returns {void}
    */
-  window._lload = function (src, callback, context) {
+  var _lload = function (src, callback, context) {
     /*eslint max-statements: [2, 20]*/
     context = context || this;
```

Comment:
I like the idea of just removing the line in question. If you remove it and make no other changes you'd effectively get this:

If nothing was passed in
`callback.apply(undefined, cb)`

if a context was passed
`callback.apply(someUserDefinedContext, cb)`

Bonus point for removing bytes too :)